src/Views/FirmwareView.vala

```diff
@@ -33,7 +33,7 @@ public class About.FirmwareView : Granite.SimpleSettingsPage {
         Object (
             icon_name: "application-x-firmware",
             title: _("Firmware"),
-            description: _("Firmware updates provided by device manufacturers can improve performance and fix critical security issues.")
+            description: _("Any new firmware is provided by respective device manufacturers, in the hopes of improving performance and fixing security issues.")
```

Comment:
This seems unnecessarily "lawyer-y" and I don't think improves clarity

Reply:
The original opens a whole unanswered question of "what else can it do?"
I really want critical security issues to get fixed, yes. I don't want "can" to be the operative word in that regard.

Reply:
It really depends on the device manufacturer. Sometimes firmware updates enable new features or are just bug fixes. It is indeed open ended what the update may or may not include